node/src/node.rs

```diff
@@ -312,7 +299,7 @@ impl<N: GossipNetwork, S: Storage, R: RawRepository> SimperbyApi for Node<N, S,
 
         let governance = t1.await?;
         let consensus = t2.await?;
-        let _ = t3.await?;
+        t3.await?;
```

Comment:
Why didn't you use join_all?

Reply:
They're spawned tasks. They will be executed concurrent whether or not it is joined together.

Reply:
right. I got it wrong.